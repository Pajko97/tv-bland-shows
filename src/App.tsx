import React from 'react';
import logo from './logo.svg';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import AppRoutes from './routes';
import style from './App.module.css'
import './assets/css/root.css'

function App(props: any) {
	
	return (
		<div>
			<BrowserRouter>
				<div>
					<div className={style.header}>
						<h1>TV Bland</h1>
					</div>
					<AppRoutes/>
				</div>
			</BrowserRouter>
		</div>

	);
}

export default App;
