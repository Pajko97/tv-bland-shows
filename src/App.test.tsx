import React from "react";
import { render, screen } from "@testing-library/react";
import App from "./App";
import renderer from "react-test-renderer";

test("renders learn react link", () => {
  render(<App />);
  const linkElement = screen.getByText("GO");
  expect(linkElement).toBeInTheDocument();
});

it("renders landing", async () => {
  let render;
  render = renderer.create(<App />).toJSON();

  expect(render).toMatchSnapshot();
});
