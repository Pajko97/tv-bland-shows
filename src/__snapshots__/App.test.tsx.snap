// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders landing 1`] = `
<div>
  <div>
    <div
      className="header"
    >
      <h1>
        TV Bland
      </h1>
      <p>
        TV Show and Web series database. Create personalized schedules. Episode guide, cast, crew and character information
      </p>
    </div>
    <div
      className="landing_wrapper"
    >
      <h1>
        Welcome to my APP
      </h1>
      <p>
        To go to list of the latest shows, please click button below
      </p>
      <a
        href="/shows"
        onClick={[Function]}
      >
        <button>
          GO
        </button>
      </a>
    </div>
  </div>
</div>
`;
